Include/pyport.h

```diff
@@ -72,10 +72,34 @@ Used in:  Py_SAFE_DOWNCAST
 
 #define PY_UINT32_T uint32_t
 #define PY_UINT64_T uint64_t
+#ifdef HAVE_STDINT_H
+#include <stdint.h>
```

Comment:
Do we really need these PY_xxx macros? Can't we use <stdint.h> in 2019?

Reply:
I'm not familiar with the requirements for writing fully portable C code, so was playing it safe. If you're happy with just using stdint I'm happy to go with that.

Reply:
Please remove this code and just use INT64_MAX directly. It's already used in Python.

INT64_MAX is already used in Include/pytime.h:

```
/* _PyTime_t: Python timestamp with subsecond precision. It can be used to
   store a duration, and so indirectly a date (related to another date, like
   UNIX epoch). */
typedef int64_t _PyTime_t;
#define _PyTime_MIN INT64_MIN
#define _PyTime_MAX INT64_MAX
```

UINT64_MAX is already used by Modules/_lzmamodule.c.